Modules/_zstd/decompressor.c

```diff
@@ -64,8 +64,8 @@ _get_DDict(ZstdDict *self)
     Py_BEGIN_CRITICAL_SECTION(self);
     if (self->d_dict == NULL) {
         /* Create ZSTD_DDict instance from dictionary content */
-        char *dict_buffer = PyBytes_AS_STRING(self->dict_content);
-        Py_ssize_t dict_len = Py_SIZE(self->dict_content);
+        char *dict_buffer = self->dict_buffer;
+        Py_ssize_t dict_len = self->dict_len;
         Py_BEGIN_ALLOW_THREADS
         self->d_dict = ZSTD_createDDict(dict_buffer,
                                         dict_len);
```

Comment:
Just inline them, as in the code below.
```suggestion
        Py_BEGIN_ALLOW_THREADS
        self->d_dict = ZSTD_createDDict(self->dict_buffer, self->dict_len);
```